Sources/FoundationExtensions/UIApplication+RCExtensions.swift

```diff
@@ -40,14 +40,4 @@ extension UIApplication {
 
 }
 
-#if DEBUG
-
-private extension UIApplication {
```

Comment:
Why was this an extension on UIApplication before?

Reply:
For no particular reason :P